plugins/client.go

```diff
@@ -0,0 +1,59 @@
+package plugins
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/docker/docker/utils"
+)
+
+const (
+	versionMimetype = "appplication/vnd.docker.plugins.v1+json"
+)
+
+func NewClient(addr string) *Client {
+	// FIXME: TLS? :scream:
```

Comment:
we shouldn't need to worry about TLS if we're using local unix sockets
and IMO we shouldn't be encouraging anything else right now
(because making remote plugins coherent has more edge cases than local ones, because the network is unreliable)